.github/workflows/sonarqube-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: SonarQube Analysis
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
Why not running his on any branch and any pull request?

We should be able to see in the pull request the GitHub action result.

Reply:
I’ve updated it, it now runs on all branches and all pull requests, so the results will show in every PR.